### ISSUE TITLE
[UX] Display the cursor as 'pointer' when hovering over buttons

### ISSUE THREAD

**Issue by ghost** [backdrop/backdrop-issues#2865]
Form buttons (like the Submit button) display the default cursor when you hover over them. I think a better UX would be to change the cursor to 'pointer' when hovering over buttons instead.

Or is there a reason for this I'm not aware of?

![peek 2017-09-26 11-56](https://user-images.githubusercontent.com/2385329/30838977-2bbb46a8-a2b2-11e7-89cb-5a81213ba04d.gif)

**Comment by BWPanda** [comment]
May be related to https://github.com/backdrop/backdrop-issues/issues/909...

**Comment by opi** [comment]
Even if I used to add `cursor:pointer` on every button or clickable element, some readings make me stop. Ex: https://medium.com/simple-human/buttons-shouldnt-have-a-hand-cursor-b11e99ca374b

TL;DR; Pointer cursors are for links, not for every clickable elements. Think about your OS buttons and interface elements. None of them have a pointer cursor.

Maybe there is still a UX issue, and button needs a better hover/focus state. Cursor is only a visual clue for mouse navigation, but keyboard navigation needs a distinct focus state too.

**Comment by alanmels** [comment]
Interesting article, @opi. However, when evaluating Backdrop I always try to imagine reaction of all those Drupal 7 website users which we hope to attract over to Backdrop. And every Drupal button has a pointer cursors on hover. Even almighty https://www.google.com does. So maybe we should look at this from the regular user's usability point?

**Comment by jenlampton** [comment]
I don't know... just because Drupal did something wrong doesn't mean we should do it wrong too. Are there any other reasons for doing it this way other than "we are all used to it being wrong"? Are there accessibility benefits, etc? What does WordPress do?

**Comment by alanmels** [comment]
@jenlampton, I've just checked WordPress has pointer everywhere on buttons. However, reading online on the subject matter additionally, I am not so sure anymore. Let's see what others say on this, if any.

**Comment by olafgrabienski** [comment]
I've also read the linked article and the follow up posts. They're really convincing, with one exception in my opinion: They don't give a reliable answer to the question if many or most users somehow expect pointer cursors for inputs because they're used to it. Anyway, I guess we're not able to answer this question either.

Personally, I notice me sometimes hesitating for a second to click an input in the Backdrop interface because in advance it doesn't 'feel' very interactive. That might be partly because I'm also used to pointer cursors everywhere. But I think a more clear indication of the hover state on the button itself would (also) help a lot to change my feeling. (At the moment, on hover the default input color changes only slightly from blue to a darker blue.)

**Comment by BWPanda** [comment]
Going to close this as 'works as designed' for now. It's not a big deal, most people are happy/used to it now, and any issues can likely be fixed by using better [hover/focus states](https://github.com/backdrop/backdrop-issues/issues/909).
